src/tests/PhysicsTest.cpp

```diff
@@ -408,3 +408,52 @@ TEST_F(PhysicsManagerTest, TestVelocityControl) {
     ASSERT_EQ(physicsManager_->getAngularVelocity(objectId), Magnum::Vector3{});
   }
 }
+
+TEST_F(PhysicsManagerTest, TestSceneNodeAttachment) {
```

Comment:
From now on, suggest using Corrade::TestSuite.

Reply:
Yes but definitely in a separate PR :) 

The new test doesn't do anything that would *strongly* benefit from switching to TestSuite, so I'm fine with keeping it in gtest and rewriting with TestSuite later.